coalib/parsing/DefaultArgParser.py

```diff
@@ -119,6 +120,26 @@ def default_arg_parser(formatter_class=None):
                                 metavar='BOOL',
                                 help='Write the logs as json to a file '
                                 'where filename is specified as argument.')
+    if parser_type == 'coala-html':
```

Comment:
Hm no. That's not the case. you might have seen that coala-html has coala
as one of the dependencies. So we can reuse stuffs from coala. IMO that's
neat. Hm tried running it locally?
On 17 Mar 2016 09:39, "AbdealiJK" <notifications@github.com> wrote:

> In coalib/parsing/DefaultArgParser.py
> <https://github.com/coala-analyzer/coala/pull/1847#discussion_r56453440>:
>
> > @@ -119,6 +120,26 @@ def default_arg_parser(formatter_class=None):
> >                                  metavar='BOOL',
> >                                  help='Write the logs as json to a file '
> >                                  'where filename is specified as argument.')
> > +    if parser_type == 'coala-html':
>
> Now this is confusing. If the coala-html is a python package of its own
> ... why is the argparser for the coala-html package here ?
> This wouldnt even be called right ? We probably want to add these in
> coala-html ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/1847/files/5ee84784b42e6f34bb63f8ad833c0d99729b8d78#r56453440>
>